snc.sh

```diff
@@ -296,7 +296,7 @@ elif [ ! -f ${OPENSHIFT_PULL_SECRET_PATH} ]; then
 fi
 
 if test -z "${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE-}"; then
-    OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="$(curl -l "${MIRROR}/${OPENSHIFT_RELEASE_VERSION}/release.txt" | sed -n 's/^Pull From: //p')"
+    OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="$(curl -L "${MIRROR}/${OPENSHIFT_RELEASE_VERSION}/release.txt" | sed -n 's/^Pull From: //p')"
```

Comment:
use explicitly `--location`, as `-l` seems to have been a typo for `-L`?

Reply:
Looks like the typo.

Reply:
`curl -L` is a lot of place. Should I use `${CURL}` instead as an alias ?

Reply:
Ah could be useful, or I'm fine with the change as it is now after your comment.

Reply:
let's start with this then 👍